packages/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.6.5
```

Comment:
Bug fix usually only needs to bump the patch, so: 0.6.4+1
https://github.com/flutter/engine/pull/16935 for more info :)

Reply:
Fixed it